api/managementpb/json/client/my_sql/my_sql_client.go

```diff
@@ -25,30 +25,30 @@ type Client struct {
 }
 
 /*
-Add adds my SQL adds my SQL service and starts several agents it automatically adds a service to inventory which is running on provided node id then adds mysqld exporter and qan mysql perfschema agents with provided pmm agent id and other parameters
+AddMixin1 adds my SQL adds my SQL service and starts several agents it automatically adds a service to inventory which is running on provided node id then adds mysqld exporter and qan mysql perfschema agents with provided pmm agent id and other parameters
 */
-func (a *Client) Add(params *AddParams) (*AddOK, error) {
+func (a *Client) AddMixin1(params *AddMixin1Params) (*AddMixin1OK, error) {
```

Comment:
That's a really bad name.

Reply:
I agree. Btw I have no idea why it renames exactly one of them. Strange bug...

Reply:
That's because there is a name clash – two methods with the same name in the same package (even if in different services). It automatically resolves this during mixin stage (among other things), but that's not something we want to use in our code.